android/libraries/rib-base/src/main/java/com/badoo/ribs/core/plugin/Plugin.kt

```diff
@@ -4,15 +4,26 @@ import android.os.Bundle
 import android.view.ViewGroup
 import androidx.lifecycle.Lifecycle
 import com.badoo.ribs.core.Node
+import com.badoo.ribs.core.Rib
 import com.badoo.ribs.core.view.RibView
 
 interface Plugin
 
+interface RibAware<T : Rib> : Plugin {
+    val rib: T
```

Comment:
fyi before node was protected field in interactor, now Rib will be public

Reply:
Good point.

Reply:
But Interactor itself will be internal to the module it lives in.

Also it was from an era where Node itself was hidden. Now it's reversed, Node is the main key player and the rest are hidden as plugins.